.appveyor.yml

```diff
@@ -51,6 +51,8 @@ shallow_clone: true
 
 environment:
   DATALAD_TESTS_SSH: 1
+  # The following is currently necessary as we test a lot of logging output in high detail:
+  DATALAD_LOG_EXC: 1
```

Comment:
This would be needed across all extensions, right?

Reply:
If they test logging output to the same extend - yes.